### DISCUSSION TITLE
Modify paths in dev / test mode possible?

### DISCUSSION THREAD

**Discussion by ghost** [tauri-apps/tauri#11884] [Q&A]
When run an app in dev or test mode, the fs-api use the fixed system paths like APPDATA (linux /home/user/.local/share/app-name), etc. 
Is it possible to set APPDATA, AUDIO, CACHE,... to own locations?

It would be easier to allow it in dev/test so we can use our fixtures in our project without copy them to the system wide locations.

**Comment by FabianLars** [comment]
On Linux you could overwrite these env vars: https://specifications.freedesktop.org/basedir-spec/latest/
On macOS overwriting the `HOME` env var should work.
On windows i don't see anything other than a windows api that requires admin permissions and changes the path permanently :/
